cmd/exporters/prometheus/prometheus.go

```diff
@@ -81,10 +81,10 @@ func (me *Prometheus) Init() error {
 		return err
 	}
 
-	if x := me.Params.GetChildContentS("global_prefix"); x != "" {
-		me.Logger.Debug().Msgf("will use global prefix [%s]", x)
-		me.globalPrefix = x
-		if !strings.HasSuffix(x, "_") {
+	if x := me.Params.GlobalPrefix; x != nil {
+		me.Logger.Debug().Msgf("will use global prefix [%s]", *x)
+		me.globalPrefix = *x
+		if !strings.HasSuffix(*x, "_") {
```

Comment:
minor but instead of de-referencing again you could say
if !strings.HasSuffix(me.globalPrefix, "_") {
since you just de`referenced the line before

Reply:
changed.